diff --git a/dist/breadcrumb-item.js b/dist/breadcrumb-item.js
index e76e0b835d602cb15f0dd11f9766bd828da12fdb..32c59a6f8e50e2aeab975681891f4279d4e9ac5b 100644
--- a/dist/breadcrumb-item.js
+++ b/dist/breadcrumb-item.js
@@ -35,6 +35,7 @@ var import_breadcrumbs = require("@react-aria/breadcrumbs");
 var import_shared_utils = require("@nextui-org/shared-utils");
 var import_react = require("react");
 var import_utils = require("@react-aria/utils");
+var link_default = require("next/link");
 function useBreadcrumbItem(originalProps) {
   const [props, variantProps] = (0, import_system.mapPropsVariants)(originalProps, import_theme.breadcrumbItem.variantKeys);
   const {
@@ -53,7 +54,7 @@ function useBreadcrumbItem(originalProps) {
   const WrapperComponent = as || "li";
   const isCurrent = !!(originalProps == null ? void 0 : originalProps.isCurrent);
   const isDisabled = originalProps == null ? void 0 : originalProps.isDisabled;
-  const Component = (originalProps == null ? void 0 : originalProps.href) && !isCurrent ? "a" : "span";
+  const Component = (originalProps == null ? void 0 : originalProps.href) && !isCurrent ? link_default : "span";
   const shouldFilterDOMProps = typeof Component === "string";
   const domRef = (0, import_react_utils.useDOMRef)(ref);
   const { itemProps } = (0, import_breadcrumbs.useBreadcrumbItem)(
diff --git a/dist/breadcrumbs.js b/dist/breadcrumbs.js
index 48fbe340c8154658d30dbeda818106d49797185e..79cda4f3e0d3f01277f821e62eaae9dd17c89222 100644
--- a/dist/breadcrumbs.js
+++ b/dist/breadcrumbs.js
@@ -52,6 +52,7 @@ var import_breadcrumbs = require("@react-aria/breadcrumbs");
 var import_shared_utils = require("@nextui-org/shared-utils");
 var import_react = require("react");
 var import_utils = require("@react-aria/utils");
+var link_default = require("next/link");
 function useBreadcrumbItem(originalProps) {
   const [props, variantProps] = (0, import_system.mapPropsVariants)(originalProps, import_theme.breadcrumbItem.variantKeys);
   const {
@@ -70,7 +71,7 @@ function useBreadcrumbItem(originalProps) {
   const WrapperComponent = as || "li";
   const isCurrent = !!(originalProps == null ? void 0 : originalProps.isCurrent);
   const isDisabled = originalProps == null ? void 0 : originalProps.isDisabled;
-  const Component = (originalProps == null ? void 0 : originalProps.href) && !isCurrent ? "a" : "span";
+  const Component = (originalProps == null ? void 0 : originalProps.href) && !isCurrent ? link_default : "span";
   const shouldFilterDOMProps = typeof Component === "string";
   const domRef = (0, import_react_utils.useDOMRef)(ref);
   const { itemProps } = (0, import_breadcrumbs.useBreadcrumbItem)(
@@ -287,7 +288,7 @@ var Breadcrumbs2 = (0, import_system4.forwardRef)((props, ref) => {
         isCurrent: isLast || child.props.isCurrent,
         ...child.props,
         key: itemKey,
-        onPress: (0, import_utils3.chain)((_a = child.props) == null ? void 0 : _a.onPress, () => onAction == null ? void 0 : onAction(itemKey))
+        onClick: (0, import_utils3.chain)((_a = child.props) == null ? void 0 : _a.onClick, () => onAction == null ? void 0 : onAction(itemKey))
       });
     });
     if (!items)
diff --git a/dist/chunk-6WHGJGLN.mjs b/dist/chunk-6WHGJGLN.mjs
index b1c0c981097d35b6857d72b7fdcee5497426f436..ead2d0153139ab871292bd7ff3c581e131bdd579 100644
--- a/dist/chunk-6WHGJGLN.mjs
+++ b/dist/chunk-6WHGJGLN.mjs
@@ -44,7 +44,7 @@ var Breadcrumbs = forwardRef((props, ref) => {
         isCurrent: isLast || child.props.isCurrent,
         ...child.props,
         key: itemKey,
-        onPress: chain((_a = child.props) == null ? void 0 : _a.onPress, () => onAction == null ? void 0 : onAction(itemKey))
+        onClick: chain((_a = child.props) == null ? void 0 : _a.onClick, () => onAction == null ? void 0 : onAction(itemKey))
       });
     });
     if (!items)
diff --git a/dist/chunk-HM6BKCEH.mjs b/dist/chunk-HM6BKCEH.mjs
index 2e2be95314e8660a06066aa1eeec096cd7f374b7..b3605e751c5ce0ae43b4a7ac30075514965826ca 100644
--- a/dist/chunk-HM6BKCEH.mjs
+++ b/dist/chunk-HM6BKCEH.mjs
@@ -9,6 +9,7 @@ import { useBreadcrumbItem as useAriaBreadcrumbItem } from "@react-aria/breadcru
 import { clsx, dataAttr, objectToDeps } from "@nextui-org/shared-utils";
 import { useMemo } from "react";
 import { mergeProps } from "@react-aria/utils";
+import Link from "next/link";
 function useBreadcrumbItem(originalProps) {
   const [props, variantProps] = mapPropsVariants(originalProps, breadcrumbItem.variantKeys);
   const {
@@ -27,7 +28,7 @@ function useBreadcrumbItem(originalProps) {
   const WrapperComponent = as || "li";
   const isCurrent = !!(originalProps == null ? void 0 : originalProps.isCurrent);
   const isDisabled = originalProps == null ? void 0 : originalProps.isDisabled;
-  const Component = (originalProps == null ? void 0 : originalProps.href) && !isCurrent ? "a" : "span";
+  const Component = (originalProps == null ? void 0 : originalProps.href) && !isCurrent ? Link : "span";
   const shouldFilterDOMProps = typeof Component === "string";
   const domRef = useDOMRef(ref);
   const { itemProps } = useAriaBreadcrumbItem(
diff --git a/dist/index.js b/dist/index.js
index 5a57bbba85389e50ae005b7cfbab148d2929a3ae..6a164be94a0c22e4ef3eac362541b734039d2dee 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -57,6 +57,7 @@ var import_breadcrumbs = require("@react-aria/breadcrumbs");
 var import_shared_utils = require("@nextui-org/shared-utils");
 var import_react = require("react");
 var import_utils = require("@react-aria/utils");
+var link_default = require("next/link");
 function useBreadcrumbItem(originalProps) {
   const [props, variantProps] = (0, import_system.mapPropsVariants)(originalProps, import_theme.breadcrumbItem.variantKeys);
   const {
@@ -75,7 +76,7 @@ function useBreadcrumbItem(originalProps) {
   const WrapperComponent = as || "li";
   const isCurrent = !!(originalProps == null ? void 0 : originalProps.isCurrent);
   const isDisabled = originalProps == null ? void 0 : originalProps.isDisabled;
-  const Component = (originalProps == null ? void 0 : originalProps.href) && !isCurrent ? "a" : "span";
+  const Component = (originalProps == null ? void 0 : originalProps.href) && !isCurrent ? link_default : "span";
   const shouldFilterDOMProps = typeof Component === "string";
   const domRef = (0, import_react_utils.useDOMRef)(ref);
   const { itemProps } = (0, import_breadcrumbs.useBreadcrumbItem)(
@@ -292,7 +293,7 @@ var Breadcrumbs2 = (0, import_system4.forwardRef)((props, ref) => {
         isCurrent: isLast || child.props.isCurrent,
         ...child.props,
         key: itemKey,
-        onPress: (0, import_utils3.chain)((_a = child.props) == null ? void 0 : _a.onPress, () => onAction == null ? void 0 : onAction(itemKey))
+        onClick: (0, import_utils3.chain)((_a = child.props) == null ? void 0 : _a.onClick, () => onAction == null ? void 0 : onAction(itemKey))
       });
     });
     if (!items)
diff --git a/dist/use-breadcrumb-item.js b/dist/use-breadcrumb-item.js
index 09933a43071dd7e3ab1b1ac996d858582a43a64c..0486b75d809017920c9290f283d0aa45641403dd 100644
--- a/dist/use-breadcrumb-item.js
+++ b/dist/use-breadcrumb-item.js
@@ -32,6 +32,7 @@ var import_breadcrumbs = require("@react-aria/breadcrumbs");
 var import_shared_utils = require("@nextui-org/shared-utils");
 var import_react = require("react");
 var import_utils = require("@react-aria/utils");
+var link_default = require("next/link");
 function useBreadcrumbItem(originalProps) {
   const [props, variantProps] = (0, import_system.mapPropsVariants)(originalProps, import_theme.breadcrumbItem.variantKeys);
   const {
@@ -50,7 +51,7 @@ function useBreadcrumbItem(originalProps) {
   const WrapperComponent = as || "li";
   const isCurrent = !!(originalProps == null ? void 0 : originalProps.isCurrent);
   const isDisabled = originalProps == null ? void 0 : originalProps.isDisabled;
-  const Component = (originalProps == null ? void 0 : originalProps.href) && !isCurrent ? "a" : "span";
+  const Component = (originalProps == null ? void 0 : originalProps.href) && !isCurrent ? link_default : "span";
   const shouldFilterDOMProps = typeof Component === "string";
   const domRef = (0, import_react_utils.useDOMRef)(ref);
   const { itemProps } = (0, import_breadcrumbs.useBreadcrumbItem)(
diff --git a/dist/use-breadcrumbs.js b/dist/use-breadcrumbs.js
index 3296ee6b801cf4d34cdc434f447ca863a164d903..30756fcf2bbab82e26e627adc326cf75f2de3dda 100644
--- a/dist/use-breadcrumbs.js
+++ b/dist/use-breadcrumbs.js
@@ -45,6 +45,7 @@ var import_breadcrumbs = require("@react-aria/breadcrumbs");
 var import_shared_utils = require("@nextui-org/shared-utils");
 var import_react = require("react");
 var import_utils = require("@react-aria/utils");
+var link_default = require("next/link");
 function useBreadcrumbItem(originalProps) {
   const [props, variantProps] = (0, import_system.mapPropsVariants)(originalProps, import_theme.breadcrumbItem.variantKeys);
   const {
@@ -63,7 +64,7 @@ function useBreadcrumbItem(originalProps) {
   const WrapperComponent = as || "li";
   const isCurrent = !!(originalProps == null ? void 0 : originalProps.isCurrent);
   const isDisabled = originalProps == null ? void 0 : originalProps.isDisabled;
-  const Component = (originalProps == null ? void 0 : originalProps.href) && !isCurrent ? "a" : "span";
+  const Component = (originalProps == null ? void 0 : originalProps.href) && !isCurrent ? link_default : "span";
   const shouldFilterDOMProps = typeof Component === "string";
   const domRef = (0, import_react_utils.useDOMRef)(ref);
   const { itemProps } = (0, import_breadcrumbs.useBreadcrumbItem)(
diff --git a/package.json b/package.json
index ca2035623798dda970363b70e05688e6e03e7f25..ec01508280b1ee170e08e2af0b83053566fa9d9b 100644
--- a/package.json
+++ b/package.json
@@ -25,20 +25,21 @@
     "url": "https://github.com/nextui-org/nextui/issues"
   },
   "peerDependencies": {
-    "react": ">=18",
-    "react-dom": ">=18",
+    "@nextui-org/system": ">=2.0.0",
     "@nextui-org/theme": ">=2.1.0",
-    "@nextui-org/system": ">=2.0.0"
+    "react": ">=18",
+    "react-dom": ">=18"
   },
   "dependencies": {
-    "@react-aria/focus": "3.17.1",
+    "@nextui-org/breadcrumbs": "file:",
+    "@nextui-org/react-utils": "2.0.16",
+    "@nextui-org/shared-icons": "2.0.9",
+    "@nextui-org/shared-utils": "2.0.7",
     "@react-aria/breadcrumbs": "3.5.13",
+    "@react-aria/focus": "3.17.1",
     "@react-aria/utils": "3.24.1",
     "@react-types/breadcrumbs": "3.7.5",
-    "@react-types/shared": "3.23.1",
-    "@nextui-org/react-utils": "2.0.16",
-    "@nextui-org/shared-utils": "2.0.7",
-    "@nextui-org/shared-icons": "2.0.9"
+    "@react-types/shared": "3.23.1"
   },
   "clean-package": "../../../clean-package.config.json",
   "module": "dist/index.mjs",
