diff --git a/dist/SSRProvider.mjs b/dist/SSRProvider.mjs
index 1d23ebaa6bbc39aae16ee10cb3bd972a04dcb53b..658a235ecf1448df7782d494b7fa330ad834d982 100644
--- a/dist/SSRProvider.mjs
+++ b/dist/SSRProvider.mjs
@@ -1,3 +1,5 @@
+"use client";
+
 import $670gB$react, {useContext as $670gB$useContext, useState as $670gB$useState, useMemo as $670gB$useMemo, useLayoutEffect as $670gB$useLayoutEffect, useRef as $670gB$useRef} from "react";
 
 /*
