diff --git a/dist/VisuallyHidden.mjs b/dist/VisuallyHidden.mjs
index 1fb224c3d9422d2c2924edc3343bcabf7f038ab3..952fac16ce7c4edb252620e1e23e272ce0c9a758 100644
--- a/dist/VisuallyHidden.mjs
+++ b/dist/VisuallyHidden.mjs
@@ -1,3 +1,5 @@
+"use client";
+
 import {mergeProps as $7JYt2$mergeProps} from "@react-aria/utils";
 import $7JYt2$react, {useState as $7JYt2$useState, useMemo as $7JYt2$useMemo} from "react";
 import {useFocusWithin as $7JYt2$useFocusWithin} from "@react-aria/interactions";
